doc/team.md

```diff
@@ -1,71 +1,61 @@
-# Team Gallery
+# PyGMT Team
 
-We are an international team dedicated to building a Pythonic API for the
-Generic Mapping Tools (GMT). Our goal is to improve GMT's accessibility for
-new and experienced users by creating user-friendly interfaces with the GMT
-C API, supporting rich display in Jupyter notebooks, and integrating with
-the PyData ecosystem.
+We are an international team dedicated to building a Pythonic API for the Generic Mapping
+Tools (GMT).
 
-All are welcome to become involved with the PyGMT project! For more information
-about how to get involved, see the {doc}`contributing`.
+All are welcome to become involved with the PyGMT project! For more information about how
+to get involved, see the {doc}`contributing`.
 
-## Distinguished Contributors
+PyGMT Distinguished Contributors are recognized for their substantial contributions to PyGMT,
+which may include code, documentation, pull request review, triaging, forum responses,
+community building and engagement, outreach, and inclusion and diversity. New Distinguished
+Contributors are selected twice per year by those listed below. Distinguished Contributors
+include current or inactive contributors and inactive maintainers, so is not meant as a means
+of conveying responsibilities. Distinguished Contributors who are also active maintainers of
+the PyGMT project have responsibilities detailed in the {doc}`maintenance`.
 
-PyGMT Distinguished Contributors are recognized for their substantial
-contributions to PyGMT, which may include code, documentation, pull request
-review, triaging, forum responses, community building and engagement,
-outreach, and inclusion and diversity. New Distinguished Contributors are
-selected twice per year by those listed below.
 
-Distinguished Contributors is not meant as a means of conveying
-responsibilities. Distinguished Contributors who are also active maintainers of
-the PyGMT project and have responsibilities detailed in the
-{doc}`maintenance` have 'Maintainer' listed below their names.
+## Founders
 
-:::::{grid} 2 3 3 4
+:::::{grid} 2 3 3 5
 
-::::{grid-item-card} Dongdong Tian
+::::{grid-item-card} Leonardo Uieda
 :margin: 0 3 0 0
```

Comment:
```suggestion
:padding: 1
```
Just tried to play with the `grid-item-card` options. Changing `:margin: 0 3 0 0` to `:padding: 1` gives a more compact layout.

|Old | New |
|---|---|
| <img width="909" alt="image" src="https://github.com/GenericMappingTools/pygmt/assets/3974108/c7ccf71b-122b-469f-9ae2-389fe55cae6d"> |<img width="931" alt="image" src="https://github.com/GenericMappingTools/pygmt/assets/3974108/84532546-8e47-4924-9a60-c0ef2fd9dd7f"> |

Reply:
Like theis as there all names fit in one line 🙂!